docs/data-sources/maintenance_windows.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Maintenance Windows"
+description: |-
+    Get the information about all Wavefront maintenance windows.
+---
+
+# Data Source: wavefront_maintenance_window_all
+
+Use this data source to get information about all Wavefront maintenance windows.
+
+## Argument Reference
+* `limit` - (Optional) Limit is the maximum number of results to be returned. Defaults to 100.
+* `offset` - (Optional) Offset is the offset from the first result to be returned. Defaults to 0.
+
+## Example Usage
+
+```hcl
+# Get the information about all external links.
+data "wavefront_maintenance_window_all" "example" {
+  limit = 10
+  offset = 0
+}
+```
+
+## Attribute Reference
+
+* `maintenance_windows` - List of all external links in Wavefront. For each external link you will see a list of attributes.
```

Comment:
List of all maintenance windows in Wavefront. For each maintenance window you will see a list of attributes.